prisma/migrations/20240619181506_add_video_url/migration.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "Cosmetic" ADD COLUMN "videoUrl" TEXT;
```

Comment:
Why do we want this both in the Cosmetic table and the `CosmeticShopItem` table?